PinpointKit/PinpointKit/Sources/BoxAnnotationView.swift

```diff
@@ -16,7 +16,12 @@ public class BoxAnnotationView: AnnotationView {
     /// The corresponding annotation.
     var annotation: BoxAnnotation? {
         didSet {
-            layer.shadowPath = annotation.flatMap(PathForDrawingBoxAnnotation)?.cgPath
+            if let annotation = annotation {
+                layer.shadowPath = self.dynamicType.path(forDrawing: annotation)?.cgPath
```

Comment:
Hmm, are you interpreting "drawing" as a noun? I read `path(forDrawing: annotation)` as "path for drawing this annotation". Perhaps there’s a better word we could use.